docs/src/modules/java/pages/actions.adoc

```diff
@@ -1,62 +1,104 @@
+<1> Any classes generated from this protobuf file will be in the `com.example.fibonacci` package.
 <2> Import the Akka Serverless protobuf annotations, or options.
 <3> Let the messages declared in this protobuf file be inner classes to the Java class `FibonacciApi`.
 <4> The protobuf option (akkaserverless.service) is specific to code-generation as provided by the Akka Serverless Maven plugin. This annotation indicates to the code-generation that an Action must be generated.
 
+[.tabset]
+Scala::
++
+[source,proto,indent=0]
+.src/main/proto/com/example/fibonacci/fibonacci.proto
+----
+include::example$scala-fibonacci-action/src/main/proto/com/example/fibonacci/fibonacci.proto[tag=actions]
+----
+<1> Any classes generated from this protobuf file will be in the `com.example.fibonacci` package.
+<2> Import the Akka Serverless protobuf annotations, or options.
+<3> The protobuf option (akkaserverless.service) is specific to code-generation as provided by the Akka Serverless Maven plugin. This annotation indicates to the code-generation that an Action must be generated.
 
 == Implementing the Action
 
-An Action implementation is a Java class where you define how each message is handled. The class
+An Action implementation is a class where you define how each message is handled. The class
 `FibonacciAction` gets generated for us based on the proto file defined above. Once the
-`FibonacciAction.java` file exist, it is not overwritten, so you can freely add logic to it.
+`FibonacciAction.java` or `FibonacciAction.scala` file exist, it is not overwritten, so you can freely add logic to it.
 `FibonacciAction` extends the generated class `AbstractFibonacciAction` which we're
 not supposed to change as it gets regenerated in case we update the protobuf descriptors.
 
 `AbstractFibonacciAction` contains all method signatures corresponding to the API of the service.
 If you change the API you will see compilation errors in the `FibonacciAction` class, and you have to
 implement the methods required by `AbstractFibonacciAction`.
 
-
+[.tabset]
+Java::
++
 [source,java,indent=0]
 .src/main/java/com/example/fibonacci/FibonacciAction.java
 ----
 include::example$java-fibonacci-action/src/main/java/com/example/fibonacci/FibonacciActionGenerated.java[tag=generated-action]
 ----
-
 <1> Extends the generated `AbstractFibonacciAction`, which extends link:{attachmentsdir}/api/com/akkaserverless/javasdk/action/Action.html[`Action` {tab-icon}, window="new"].
 <2> A `nextNumber` method is generated. We will implement it next.
 
+[.tabset]
+Scala::
++
+[source,scala,indent=0]
+.src/main/scala/com/example/fibonacci/FibonacciAction.scala
+----
+include::example$scala-fibonacci-action/src/main/scala/com/example/fibonacci/FibonacciActionGenerated.scala[tag=generated-action]
+----
+<1> Extends the generated `AbstractFibonacciAction`, which extends link:{attachmentsdir}/api/com/akkaserverless/scalasdk/action/Action.html[`Action` {tab-icon}, window="new"].
```

Comment:
api links for Scala should be `{attachmentsdir}/scala-api/`

I added the rsync in the makefile: https://github.com/lightbend/akkaserverless-java-sdk/pull/514

Reply:
Thanks